src/TextField/TextField.jsx

```diff
@@ -455,6 +461,7 @@ const TextField = React.createClass({
       onKeyDown: this._handleInputKeyDown,
     };
     const inputStyle = this.mergeStyles(styles.input, this.props.inputStyle);
+    const textareaStyle = this.mergeStyles(styles.textarea, this.props.textareaStyle);
```

Comment:
Could we use the destructuration to get the `textareaStyle`? That would be simpler to follow.
```js
const textareaStyle = this.mergeStyles(styles.textarea, textareaStyle);
```